src/containers/org/components/AddMemberModalBody.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import React from 'react';
+
+import { Typography } from 'lattice-ui-kit';
+
+import SearchMemberBar from './SearchMemberBar';
+
+import { ModalBody } from '../../../components';
+
+type Props = {
+  onChange :() => void;
+};
+
+const AddMemberModalBody = ({ onChange } :Props) => (
+  <ModalBody>
+    <Typography>Enter the username or email address of the person you wish to add to the organization</Typography>
```

Comment:
The "or email" bit requires a follow-up:

https://jira.openlattice.com/browse/APPS-2508